### PR TITLE
Improve software section

### PR DESCRIPTION
**Pull request by ryw**

### CODE REVIEW

File: src/content/docs/product/software/tembo-enterprise/overview.mdx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+title: Tembo Enterprise
+sideBarPosition: 201
```
Comment (DarrenBaldwin07) [inline src/content/docs/product/software/tembo-enterprise/overview.mdx:3]: We need to do this for all other docs but for new ones we should get in habit of writing a `description: ` if we can for SEO

**Review by DarrenBaldwin07 — APPROVED** [review]

File: src/content/docs/product/software/tembo-enterprise/overview.mdx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	we also provide software for various use cases.
+</Callout>
+
+
+## Tembo Enterprise
+
+In the Enterprise Software model, both the control plane and the data plane are 
+owned and managed by you, and will live in your cloud account. 
+
+We'll attempt to provide you with a way to install and run the control plane,
```
Comment (ianstanton) [inline src/content/docs/product/software/tembo-enterprise/overview.mdx:19]:
```suggestion
We'll provide you with a way to install and run the control plane, 
```

File: src/content/docs/product/software/tembo-enterprise/overview.mdx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+We'll attempt to provide you with a way to install and run the control plane, 
+data plane and UI in your account so that they can use Tembo Cloud in a 
+way that meets your security requirements. 
+
+Extension management could be a challenge here. You may not be able to use the 
+public Trunk registry, so we may need to provide a way for you to use your 
+own private extension registry.
+
+<img src="/enterprise-software.png" />
```
Comment (ianstanton) [inline src/content/docs/product/software/tembo-enterprise/overview.mdx:27]: I'll clean this image up with a more generic one. It should also show all components in one Kubernetes cluster

File: src/content/docs/product/software/tembo-enterprise/overview.mdx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+* General support
+
+Your Responsibilities: 
+
+* Kubernetes cluster provisioning
+* Data plane component installation
+* Data plane connection to Tembo's control plane
+* Data plane component upgrades, maintenance, and support
+* Kubernetes cluster upgrades, maintenance, and support
```
Comment (ianstanton) [inline src/content/docs/product/software/tembo-enterprise/overview.mdx:42]: We should clean this up as it suggests a hybrid approach

File: src/content/docs/product/software/tembo-enterprise/overview.mdx [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+In the Enterprise Software model, both the control plane and the data plane are 
+owned and managed by you, and will live in your cloud account. 
+
+We'll attempt to provide you with a way to install and run the control plane, 
+data plane and UI in your account so that they can use Tembo Cloud in a 
+way that meets your security requirements. 
+
+Extension management could be a challenge here. You may not be able to use the 
+public Trunk registry, so we may need to provide a way for you to use your 
+own private extension registry.
```
Comment (ianstanton) [inline src/content/docs/product/software/tembo-enterprise/overview.mdx:25]:
```suggestion
Extension management is a key feature of the Tembo experience, and the platform makes use of the Trunk extension registry under the hood. You may not be able to use the 
public Trunk registry, so we'll provide a way for you to use your 
own private extension registry.
```

**Pull request merged**